app/models/grda_warehouse/recurring_hmis_export.rb

```diff
@@ -220,7 +220,10 @@ def self.available_s3_regions
 
     def filter_hash
       hash = options
+      hash[:reporting_range] = reporting_range
+      hash[:reporting_range_days] = reporting_range_days
       hash[:recurring_hmis_export_id] = id
+      hash[:version] ||= '2024'
```

Comment:
This is fine. Maybe someday we can extract magic numbers into constants or centralized config

Reply:
Yeah, I almost left this with a `fixme`, but it's only used for existing recurring export configs, anything that gets added gets the version appropriately added.